RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectrons_cfi.py

```diff
@@ -1,10 +1,10 @@
 from RecoEgamma.EgammaElectronProducers.gsfElectrons_cfi import ecalDrivenGsfElectrons
 
-lowPtGsfElectrons = ecalDrivenGsfElectrons.clone(gsfElectronCoresTag = "lowPtGsfElectronCores")
+lowPtGsfElectronsPreRegression = ecalDrivenGsfElectrons.clone(gsfElectronCoresTag = "lowPtGsfElectronCores")
```

Comment:
the file should be renamed to `lowPtGsfElectronsPreRegression_cfi.py`

Reply:
Done.